### PR TITLE
WIP New module: manage VyOS vlans (network/vyos/vyos_vlan)

### PR DESCRIPTION
**Pull request by rcarrillocruz**

### CODE REVIEW

File: lib/ansible/modules/network/vyos/vyos_vlan.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  state:
+    description:
+      - State of the VLAN configuration. 
+    default: present
+    choices: ['present', 'absent', 'active', 'suspend']
+"""
+
+EXAMPLES = """
+- name: configure hostname and domain name
+  net_system:
```
Comment (ganeshrn) [inline lib/ansible/modules/network/vyos/vyos_vlan.py:64]: Should the example be for net_vlan?

File: lib/ansible/modules/network/vyos/vyos_vlan.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python
```
Comment (gundalow) [inline lib/ansible/modules/network/vyos/vyos_vlan.py:1]: Should this be `network/net_vlan.py`

Comment (gundalow) [reply lib/ansible/modules/network/vyos/vyos_vlan.py:1]: Does `lib/ansible/plugins/action/net_vlan.py` need adding to this PR?

File: lib/ansible/modules/network/vyos/vyos_vlan.py (hunk 1 above)
Comment (rcarrillocruz) [reply lib/ansible/modules/network/vyos/vyos_vlan.py:64]: yes, it's WIP at the moment

**Pull request closed without merging**